### ISSUE TITLE
Problem starting Redirection 5.3.10

### ISSUE THREAD

**Issue by isajk** [johngodley/redirection#3643]
Redirection stopped working recently. REST is fine, I can load the javascript, caching is not an issue. I just tried the plugin on FireFox and it took me through a setup. However that setup seems to have wiped all the redirects that I have set up. I have a real estate website. As properties sell, I redirect the property's page to a SOLD page. Now the redirects are all gone. 

Also even though it works in Firefox, it still doesn't work in old Safari.

```
SyntaxError: Unexpected token '.' https://txhorseproperties.com/2016/wp-content/plugins/redirection/redirection.js 2
```

Plugin: 5.3.10 4.2
WordPress: 6.2.2 (single)
PHP: 7.4.33
Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_6) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/12.1.2 Safari/605.1.15
JavaScript: https://txhorseproperties.com/2016/wp-content/plugins/redirection/redirection.js?ver=5.3.10-948a20e46d8617ec7b6cf37114495e49
REST API: https://txhorseproperties.com/wp-json/

**Comment by johngodley** [comment]
Why did it stop working suddenly? Unless the plugin changed (it hasn't) then something else must have changed on your site. As it says on the error page the most likely cause is another plugin.

> it works in Firefox,

So it does work?

**Comment by isajk** [comment]
I disabled the plugins and it does not work via Safari at all. It displays the error page with the recommendations for checking the javascript and REST api and disabling the plugins. It is an old version of Safari (12.1.2), but it has worked for a long time (years?). After a recent update, it failed. I tried re-installing an earlier version, but that did not work either. 

While it does work with Firefox, all the redirections that had previously been created are gone. I did not delete them, but running the update via Firefox yesterday seems to have created a new version with none of the previous redirects.
